docker/README.md

```diff
@@ -0,0 +1,112 @@
+
```

Comment:
docker in a separated section or including in the different section of unit tests or acceptance tests?